cloudbuild.yml

```diff
@@ -77,6 +77,13 @@ steps:
   args: ['./scripts/run-build.sh', 'universal-sentence-encoder']
   waitFor: ['diff']
 
+# Handpose
+- name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: 'bash'
+  id: 'handpose'
+  args: ['./scripts/run-build.sh', 'handpose']
+  waitFor: ['diff']
+
```

Comment:
Could we add facemesh here as well, while you're touching this file?

Reply:
done